3scale/3scale-operator
# MGDAPI-3467 tenant deletion support

- adds support of deleting tenants via CR
  * https://issues.redhat.com/browse/MGDAPI-3467
  * https://issues.redhat.com/browse/THREESCALE-6051

## Verification:
For RHOAM:
- install RHOAM locally from this branch: https://github.com/integr8ly/integreatly-operator/pull/2409 but replace the manifests/3scale/0.9.0 values for containerImage and image from `quay.io/austincunningham/3scale-operator:cluster-scoped` to `quay.io/mstoklus/3scale-operator:v0.12.0`
- wait for install to complete 
- create tenant via CR
- confirm tenant got created in 3scale 
- delete tenant CR
- confirm tenant has been marked for deletion in 3scale
